memory/src/main/scala/filodb.memory/format/vectors/HistogramVector.scala

```diff
@@ -102,13 +106,16 @@ object BinaryHistogram extends StrictLogging {
   val HistFormat_Null = 0x00.toByte
   val HistFormat_Geometric_Delta = 0x03.toByte
   val HistFormat_Geometric1_Delta = 0x04.toByte
+  val HistFormat_OtelExp_Delta = 0x09.toByte
   val HistFormat_Custom_Delta = 0x05.toByte
   val HistFormat_Geometric_XOR = 0x08.toByte    // Double values XOR compressed
   val HistFormat_Custom_XOR = 0x0a.toByte
 
-  def isValidFormatCode(code: Byte): Boolean =
+  def isValidFormatCode(code: Byte): Boolean = {
     (code == HistFormat_Null) || (code == HistFormat_Geometric1_Delta) || (code == HistFormat_Geometric_Delta) ||
-    (code == HistFormat_Custom_Delta)
+    (code == HistFormat_Custom_Delta) || (code == HistFormat_OtelExp_Delta)
+    // Question: why are other formats not here as valid ?
```

Comment:
Very valid and interesting point. I am taking a look offline and we will discuss this more

Reply:
Do you mean the other otel histograms ?

Reply:
No those like `HistFormat_Geometric_XOR`. This is from code by prior authors that I am not understanding yet.